### ISSUE TITLE
[MHV Medications / Accessibility] List of medications provided without list elements

### ISSUE THREAD

**Issue by sarahhortonadhoc** [department-of-veterans-affairs/va.gov-team#124861]
### 📝 User story

As a screen reader user, I want the list of medications that are taking longer to refill to be provided as a list, so that my screen reader tells me how many medications are in the list and which medication I'm on as I navigate through the list.

### 📝 Problem statement

Each medication on the list is provided using the paragraph (`<p>`) element.

### 🌌 Supporting screenshots or videos

<img width="477" height="401" alt="Screenshot of alert with a heading a list of links to medications" src="https://github.com/user-attachments/assets/3e7125c5-82e5-4575-96a9-8d418a930347" />

`<p …><a … href="/my-health/medications/prescription/25567989">ALBUTEROL 0.5% INHL SOLN</a></p>`

### 👣 Reproduction steps

1. Navigate to the [Medications list page](https://staging.va.gov/my-health/medications?page=1) (username: [karsyn.gpsysasynctwo@id.me](mailto:karsyn.gpsysasynctwo@id.me))
2. Review the underlying code in the Alert.
3. Notice the use of `<p>` elements.
4. Using a screen reader, use the Tab key to navigate to the first item in the medications links list.
5. Notice that the screen reader does not announce details about the number of items in the list and the item number of the focused link.

### 🔗 Point of contact

@sarahhortonadhoc

### ⚠️ Severity level

3, Moderate. Should be fixed in 1-3 sprints post-launch.

#### Accessibility Standards
- [Guideline 1.3 Adaptable: Create content that can be presented in different ways (for example simpler layout) without losing information or structure.](https://www.w3.org/WAI/WCAG22/Understanding/adaptable)
  - [1.3.1 Info and Relationships (Level A)](https://www.w3.org/WAI/WCAG22/Understanding/info-and-relationships.html)

#### Experience Standards
- [Category Usability: User is able to easily know or learn how to use the interface to complete tasks.](https://design.va.gov/about/experience-standards/#category-usability)
  - [User can identify an element and its state](https://design.va.gov/about/experience-standards/#user-can-identify-an-element-and-its-state)

### 𝌣 Proposed solution or next steps

Mark up the list of medications in the Alert as an unordered list, with `<ul>` and `<li>` elements.

### 𝌣 Supporting resources
- [List component](https://design.va.gov/components/list) — from VADS

### ✅ Acceptance criteria
- [ ] The system provides lists using list elements.
- [ ] The screen reader user hears list details when navigating the list of links.

### ✅ Possible tasks

- [x] Review, refine, and acknowledge feedback.
- [x] Update ticket information (title, assignee, labels, project(s), etc.)
- [ ] Fix and/or document decisions made.
- [ ] Product manager to move ticket in their respective product board to 'QA/A11y Review' after reviewing documented decisions / validating fix.
- [ ] Accessibility specialist will close the ticket after reviewing documented decisions / validating the fix.

**Comment by sarahhortonadhoc** [comment]
See PR: [Refactor Medications List to use semantic html #40371](https://github.com/department-of-veterans-affairs/vets-website/pull/40371)
